packages/elements/src/components/ino-img/ino-img.tsx

```diff
@@ -138,6 +144,10 @@ export class Image implements ComponentInterface {
     this.composedRatioHeight = `${ratio}%`;
   }
 
+  private handleError() {
+    this.el.querySelector('.ino-img__fallback-icon').setAttribute("style", "display: block");
```

Comment:
Instead of using inline styles, we should add a class to the element which displays the fallback image (e.g. `ino-img__fallback-icon--show`).

Reply:
✅